nuxeo-core/nuxeo-core-schema/src/main/resources/OSGI-INF/SchemaService.xml

```diff
@@ -156,13 +156,44 @@
       Extension Point is also used to register additional property information such as:
       <ul>
         <li>secured: only administrators can edit it</li>
+        <li>deprecated/removed: enable deprecation mechanisms for this property</li>
```

Comment:
Maybe rework the documentation to state that the information is the `deprecation` attribute that can be `deprecated` or `removed`?